components/image.html

```diff
@@ -0,0 +1,218 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>POOL.ui | Saurabh Kamboj</title>
+    <link rel="shortcut icon" href="/images/favicon-32x32.png" type="image/png">
+
+    <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="/mobile.layout.style.css">
+    <link rel="stylesheet" href="/components/image.css">
+    <link rel="stylesheet" href="/prism.css">
+</head>
+
+<body>
+
+    <div class="sidebar" id="side-nav">
+        <img  class="close-icon" src = "/images/cross.svg">
+        <div class="sidebar-content">
+            <a href="/index.html" class="sidebar-heading">Home</a>
+            <a href="/component.html" class="sidebar-heading">Components</a>
+            <div class="sidebar-block">
+                <a href="/components/alert.html" >Alert</a>
+                <a href="/components/avatar.html" >Avatar</a>
+                <a href="/components/badge.html" >Badge</a>
+                <a href="/components/button.html" >Button</a>
+                <a href="/components/card.html" >Card</a>
+                <a href="/components/image.html" >Image</a>
+                <a href="/components/input.html" >Input</a>
+                <a href="/components/list.html" >List</a>
+                <a href="/components/modal.html" >Modal</a>
+                <a href="/components/navigation.html" >Navigation</a>
+                <a href="/components/slider.html">Slider</a>
+                <a href="/components/text_utilities.html" >Text-Utilities</a>
+                <a href="/components/toast.html" >Toast</a>
+            </div>
+            <div class="space20"></div>
+        </div>
+    </div>
+
+    <div class="flex-container">
+
+        <div class="flex-header">
+            <img class="open-icon" src="/images/open.svg">
+            <div class="flex-head">POOL.ui</div>
+            <div class="flex-author">| &nbsp;&nbsp; Saurabh Kamboj </div>
+        </div>
+
+        <div class="component-container">
+            <div class="component-heading">Image</div>
+
+            <div class="component-block block-odd">
+                <div class="component-block-heading">Rounded Images</div>
+                <div class="component-block-item display-flex"> 
+
+                    <img class="img-rounded img50x50" src= "/components/component_img/img.jpg" />
```

Comment:
Please include the **ALT** tag inside the img tag as it is consider the best practice to include it for web accessiblity.
I read somewhere that it is used for screen-reader and also provide the contextual description of the image when it fails to load.